recipes/pauda-mergeblast/merge.blast.pl

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env perl
+# xiechaos@gmail.com
```

Comment:
Correct, its not needed. It is a complementary script for merging BLAST files produced by PAUDA.